.github/CHANGELOG.md

```diff
@@ -107,6 +107,9 @@
 
 ### Bug fixes
 
+* Add concept restriction to ensure `ConstMult` inline function only hit with arithmetic-values times complex values. Fixes build failures with +Test+OpenMP~BLAS~Python under clang.
```

Comment:
This doesn't render nicely. Will you update the format of `+Test+OpenMP~BLAS~Python`?

Reply:
Done.